lib/spack/spack/bootstrap.py

```diff
@@ -785,7 +785,10 @@ def ensure_gpg_in_path_or_raise():
 
 def patchelf_root_spec():
     """Return the root spec used to bootstrap patchelf"""
-    return _root_spec('patchelf@0.13:')
+    # TODO: patchelf is restricted to v0.13 since earlier versions have
+    # TODO: bugs that we don't to deal with, while v0.14 requires a C++17
+    # TODO: which may not be available on all platforms.
+    return _root_spec('patchelf@0.13:0.13.1')
```

Comment:
```suggestion
    return _root_spec('patchelf@0.13.1:0.13')
```

Reply:
Would you be fine with `.99`? :upside_down_face:

Reply:
sure, but do note that #27681 is required

Reply:
So, my original idea was to allow 0.13 and 0.13.1 Is there any super relevant bug for which we need to use 0.13.1 only? That said I plan to merge the 2 PRs in sequence.

Reply:
0.13 has multiple bugs, so if you want something stable, require 0.13.1

Reply:
https://github.com/spack/spack/pull/27675 could be closed if we require 0.13.1. Although arguably it doesn't make any sense to set an empty rpath

Reply:
Ok, done